.github/CODEOWNERS

```diff
@@ -1,7 +1,44 @@
+# Default group
+*                                                       @Cray-HPE/docs-csm-reviewers
 
-*                       @Cray-HPE/docs-csm-reviewers
+# Metal team items.
+background/ncn_*                                        @Cray-HPE/metal
+install/*                                               @Cray-HPE/metal
+install/livecd                                          @Cray-HPE/metal
+operations/bare_metal                                   @Cray-HPE/metal
 
-background/ncn_*        @Cray-HPE/metal
-install/*               @Cray-HPE/metal
-install/livecd          @Cray-HPE/metal
-operations/network      @Cray-HPE/management-network
+# Continuous Integration Items
+Jenkinsfile.github                                      @Cray-HPE/CI
+
+# CMS Items
+operations/boot_orchestration                           @Cray-HPE/cms-core-bos
```

Comment:
It's best to have a slash at the beginning of each of these lines.  That will make it specifically for the operations directory (for example) at the root of the repository.  Otherwise it will be for any operations directory in docs-csm.